### PR TITLE
chore: [CHK-3275] init

### PR DESCRIPTION
**Pull request by infantesimone**
The goal of this PR is to initialize the microservice.

### CODE REVIEW

File: src/main/resources/application.properties
Comment (scaminati-bv) [inline src/main/resources/application.properties:1]: On others repositories application name was set as a variable like this:

application.name=${name}
build.version=${version}

I assume this configurations will be set later.

**Review by CianoDanilo — APPROVED** [review]

File: eclipse-style.xml
Comment (EmanueleBVtech) [inline eclipse-style.xml:1]: Maybe eclipse specific configuration files should maybe removed from the repo using the .gitignore?
Just to keep your repository clean and avoid unnecessary clutter.

File: src/main/resources/application.properties
Comment (infantesimone) [reply src/main/resources/application.properties:1]: why not in this PR :-) done

File: eclipse-style.xml
Comment (infantesimone) [reply eclipse-style.xml:1]: used by spotless plugin

**Pull request merged**